spec/typinggame_server/interactors/players/fetch_all_players_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Interactors::Players::FetchAllPlayers do
+  let(:repository) { PlayerRepository.new }
+  let(:interactor) { described_class.new(repository: repository) }
+
+  before do
+    repository.clear
+
+    repository.create(id: '1')
+  end
+
+  it 'fetches all players' do
+    expect(interactor.call.players).to eq(repository.all)
```

Comment:
Would be nice if this test assertion wasn't a database query. Being explicit it preferred.

Reply:
@jgayfer fixed.